data/campaigns/Sceptre_of_Fire/scenarios/2t_In_the_Dwarven_City.cfg

```diff
@@ -149,18 +155,17 @@
             speaker=Durstorn
             message= _ "Good one, let them try to take it! Ha!"
         [/message]
+        [unhide_unit]
+            id=Rugnur
+        [/unhide_unit]
         {MOVE_UNIT id=Rugnur 8 10}
-        [recall]
-            id=Baglur
-            x,y=9,9
-        [/recall]
         [message]
             speaker=Rugnur
-            message= _ "Hey, Durstorn, have we started yet? King Haldric came by just now, wanted to talk to you..."
+            message= _ "Hey, Durstorn, have we started yet?"
         [/message]
         [message]
             speaker=Durstorn
-            message= _ "Speak properly to your elders, boy! Why are you late?"
+            message= _ "Speak properly to your elders, boy! What are you taking about?  Why are you late?"
```

Comment:
"What are you taking about?  Why are you late?"
--> "What are you talking about? Why are you late?"
('talking' and only a single space)